translate/src/modules/search/components/SearchBox.tsx

```diff
@@ -278,6 +291,13 @@ export function SearchBoxBase({
         setTimeRange={setTimeRange}
         updateFiltersFromURL={updateFiltersFromURL}
       />
+      <SearchPanel
+        filters={filters}
+        resetFilters={resetFilters}
+        applyFilters={applyFilters}
+        applySingleFilter={applySingleFilter}
+        toggleFilter={toggleFilter}
```

Comment:
Why do we need all these Filters data and logic in the search options? It's a separate panel, so I would expect it to store data separately.